### PR TITLE
jpegview: Add persistence

### PR DESCRIPTION
**Pull request by dodorz**

### CODE REVIEW

File: bucket/jpegview.json [hunk 1]
```diff
@@ -24,6 +25,11 @@
         "url": "https://sourceforge.net/projects/jpegview/rss?path=/",
         "re": "/jpegview/([\\d.]+)/JPEGView_"
     },
+    "persist": [
+        "JPEGView.ini",
+        "KeyMap.txt",
+        "ParamDB.db"
```
Comment (Ash258) [inline bucket/jpegview.json]: Where did you get information about this file?

When it is created it is always 0bytes and also when you delete it application do not create it.

Comment (Ash258) [reply bucket/jpegview.json]: A

Comment (Ash258) [reply bucket/jpegview.json]: You also need to copy old folder to persist for backward compatability

Comment (Ash258) [reply bucket/jpegview.json]: If you just create it then it cannot be used

A

See ditto manifest how to handle this

**Comment by issaclin32** [comment]
I think **manual persist** (in pre_install/uninstaller) is the right way to do this, because an empty `ParamDB.db` cannot work. Also whether `ParamDB.db` exists depends on user settings.

**Comment by rashil2000** [comment]
The `.db` file can be created like this:

https://github.com/ScoopInstaller/Extras/blob/1cd0ac886e13c86efbded46dedc068be42c8717d/bucket/ditto.json#L18-L26

**Comment by issaclin32** [comment]
> The `.db` file can be created like this:

Thanks for the info, but `ParamDB.db` will only be created when user clicks `right-click -> save parameter to DB`. So this approach cannot work.

The usage of ParamDB can be found at: http://wa2l-wintools.sourceforge.net/man1/jpegview.Readme.1.html

**Comment by issaclin32** [comment]
Superseded by #7632

**Pull request closed without merging**